ipaserver/install/kra.py

```diff
@@ -45,15 +45,15 @@ def install_check(api, replica_config, options):
             raise RuntimeError(
                 "Dogtag CA is not installed.  Please install the CA first")
 
+    if options.promote:
```

Comment:
You should still rather keep this check in the `if replica_config is not None` branch